graph/patterns/change-tracking.md

```diff
@@ -0,0 +1,161 @@
+# Change tracking
+
+Microsoft Graph API Design Pattern
+
+*The change tracking pattern provides the ability for API consumers to request changes in data from Microsoft Graph without having to re-read data that has not changed.*
+
+
+## Problem
+
+API consumers require an efficient way to acquire changes to data in the Microsoft Graph, for example to synchronize an external store or to drive a change-centric business process.
+
+## Solution
+
+API designers can enable the change tracking (delta) capability on a resource in the Microsoft Graph (typically on an entity collection or a parent resource) by declaring a delta function on that resource.
+
+This function returns a delta payload. A delta payload consists of a collection of annotated full or partial Microsoft Graph entities plus either a `nextLink` to further pages of original or change data that are immediately available OR a `deltaLink` to poll to get the next set of changes as they occur.
+
+Annotations allow the delta payload to indicate resources or links which have been deleted. API callers are expected to differentiate resource adds from updates by interpreting the id property of the change records against the existence of resources in whatever external system is doing the processing.
+
+The pattern requires a sequence of requests.
```

Comment:
The pattern bellow doesn't seem correct.  Please see the details here: https://learn.microsoft.com/en-us/graph/delta-query-overview?tabs=http#use-delta-query-to-track-changes-in-a-resource-collection

Reply:
Maybe I'm missing something, but I believe it's very close: 1. delta no param, 2. optional if there is nextLink 3. update on delta with delta token 4. optional if there is nextLink. added a link to pub docs

Reply:
@dkershaw10 could you please elaborate on what is the main deviation besides ability to use latest?